src/main/java/com/webdev/cosmo/cosmobackend/security/Sample.java

```diff
@@ -0,0 +1,18 @@
+package com.webdev.cosmo.cosmobackend.security;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class Sample {
+    @GetMapping(path = "/user")
```

Comment:
I suggest stick with consistency, and when You are using path ="" (which You do not have to, as it is defualt param), use it everywhere. But i'd say, do not use it if it is the only param

Reply:
This comment is for the code quality, do not fix, just bear in mind